fusion-cli/build/get-webpack-config.js

```diff
@@ -398,12 +395,29 @@ function getWebpackConfig(opts /*: WebpackConfigOpts */) {
       runtime === 'server' &&
         ((context, request, callback) => {
           if (/^[@a-z\-0-9]+/.test(request)) {
-            // do not bundle external packages and those not whitelisted
             const absolutePath = resolveFrom.silent(context, request);
+            // do not bundle external packages and those not whitelisted
             if (absolutePath === null) {
               // if module is missing, skip rewriting to absolute path
               return callback(null, request);
             }
+            if (experimentalCompileTest) {
+              const {bundle} = experimentalCompileTest(absolutePath, {
+                transform: 'none', // default transform value doesn't actually matter here
```

Comment:
Shouldn't the default be based on whether in `node_modules` vs `src/`?

Reply:
Oh, I see this function is potentially called twice for each module. Would this be confusing for users? I wonder if we should store the value and reference it in the babel check.

Reply:
From my understanding, that is done via the regex test on line 397. I thought this code branch was already running on only node_modules. 

However, now that I think about it, I think that behavior might change now that we effectively support bundling some node_modules... we need to change that logic.

Reply:
We could also update the API to include the environment, which would allow different configurations for server vs browser.

Reply:
Also, wouldn't returning 'none' for babel result in babel not parsing that file at all? So it seems like it could be useful for performance to have different config for server vs browser here